docker/gemini/Dockerfile

```diff
@@ -5,9 +5,9 @@ WORKDIR /gemini
 RUN apk add curl
 
 ARG version
-ENV version=${version:-1.7.6}
+ENV version=${version:-1.8.7}
 
-RUN curl -LO https://github.com/scylladb/gemini/releases/download/v${version}/gemini_${version}_Linux_x86_64.tar.gz ;\
+RUN curl -LO https://github.com/scylladb/gemini/releases/download/v${version}/gemini_${version}_linux_amd64.tar.gz ;\
```

Comment:
Note the change in the filename. Unsure if we actually use this Dockerfile, btw!